mlflow/cli.py

```diff
@@ -554,6 +564,24 @@ def gc(older_than, backend_store_uri, run_ids):
     else:
         run_ids = run_ids.split(",")
 
+    deleted_experiment_ids = []
+    current_time = int(time.time() * 1000)
```

Comment:
We have a function to get the current time in milliseconds here: https://github.com/mlflow/mlflow/blob/b26eb133f96fd0cfd4833546490730367ccef46f/mlflow/utils/time_utils.py#L15

Reply:
Yes, I did.